cypress/integration/purchase/purchase_order_from_sales_order.js

```diff
@@ -0,0 +1,136 @@
+import { PackingInstructions } from '../../support/utils/packing_instructions';
+import { PackingInstructionsVersion } from '../../support/utils/packing_instructions_version';
+import { salesOrders } from '../../page_objects/sales_orders';
+import { Builder } from '../../support/utils/builder';
+
+describe('Create Purchase order - material receipt - invoice', function() {
+  const timestamp = new Date().getTime();
+  const productForPackingMaterial = `ProductPackingMaterial ${timestamp}`;
+  const productPMValue = `purchase_order_testPM ${timestamp}`;
+  const packingMaterialName = `ProductPackingMaterial ${timestamp}`;
+  const packingInstructionsName = `ProductPackingInstrutions ${timestamp}`;
+  const productName1 = `ProductTest ${timestamp}`;
+  const productValue1 = `purchase_order_test ${timestamp}`;
+  const productCategoryName = `ProductCategoryName ${timestamp}`;
+  const productCategoryValue = `ProductCategoryValue ${timestamp}`;
+  const discountSchemaName = `DiscountSchemaTest ${timestamp}`;
+  const priceSystemName = `PriceSystem ${timestamp}`;
+  const priceListName = `PriceList ${timestamp}`;
+  const priceListVersionName = `PriceListVersion ${timestamp}`;
+  const productType = 'Item';
+  const vendorName = `Vendor ${timestamp}`;
+  const customerName = `Customer ${timestamp}`;
+
+  before(function() {
+    Builder.createBasicPriceEntities(priceSystemName, priceListVersionName, priceListName, true);
+    cy.fixture('discount/discountschema.json').then(discountSchemaJson => {
+      Object.assign(new DiscountSchema(), discountSchemaJson)
+        .setName(discountSchemaName)
+        .apply();
+    });
+    /**Create product for packing material */
+    Builder.createBasicProductEntitiesWithPrice(priceListName, productForPackingMaterial, productPMValue, productType);
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialName)
+        .setProduct(productForPackingMaterial)
+        .apply();
+    });
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsName)
+        .apply();
+    });
+    cy.fixture('product/packing_instructions_version.json').then(pivJson => {
+      Object.assign(new PackingInstructionsVersion(), pivJson)
+        .setName(packingInstructionsName)
+        .setPackingInstructions(packingInstructionsName)
+        .setPackingMaterial(packingMaterialName)
+        .apply();
+    });
+    cy.fixture('product/simple_productCategory.json').then(productCategoryJson => {
+      Object.assign(new ProductCategory(), productCategoryJson)
+        .setName(productCategoryName)
+        .setValue(productCategoryValue)
+        .apply();
+    });
+    /**Create vendor to use in product - Business partner tab */
+    new BPartner({ name: vendorName })
```

Comment:
Doesn't it make sense to use fixture for bpartners as well ?

Reply:
Should I create a fixture for vendor? as currently we have only for customer.

Reply:
Yeah, why not.

Reply:
Done, I haven't obtained a stable build but  on the last run: https://dashboard.cypress.io/#/projects/5yp4q1/runs/356/specs - I had only 2 tests which are failing and I don't think it's because of my test.  I couldn't obtain a stable build on this branch; I have executed the tests on 3 different instances ( dev596, dev595, dev591) - I had 3 different issues which are failing but I don't think it's related to the changes I did for this test. I need your advice on this.